Makefile

```diff
@@ -1,23 +1,45 @@
 test:
 	tox
 
-build:
-	rm -f dist/* && python3 -m pip install --upgrade build && python3 -m build .
+.PHONY: build
+build: setup_dependencies
+	rm -rf dist/* && poetry build && ./rename-wheel.sh
+
+install: setup_dependencies
+	poetry install --with dev
+
+.ONESHELL:
```

Comment:
Maybe add a comment about why this is needed (is it that poetry config sets env vars or somehthing?)

Reply:
Actually I think this is probably unnecessary after removing the poetry installation from the `Makefile` - it was originally so that the `install poetry -> add plugin` worked end to end but it's not there anymore.